org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/ScanInformation.java

```diff
@@ -127,6 +128,12 @@ public void setScannableNames(Collection<String> scannableNames) {
 		shape = estimator!=null ? estimator.getShape() : null;
 		return shape;
 	}
+	
+	public long getEstimatedScanTime() {
+		if (estimatedScanTime > 0) return estimatedScanTime;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)